Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -103,16 +103,22 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
     public func correctFile(file: File) -> [Correction] {
         let matches = violationRangesInFile(file, withPattern: pattern)
         guard !matches.isEmpty else { return [] }
+        let fileregions = file.regions()
 
         let regularExpression = regex(pattern)
         let description = self.dynamicType.description
         var corrections = [Correction]()
         var contents = file.contents
         for range in matches.reverse() {
+            let location = Location(file: file, characterOffset: range.location)
+            let region = fileregions.filter {$0.contains(location)}.first
+            if region?.isRuleDisabled(self) == true {
```

Comment:
tests fail. when region is nil it doesn't fire but we need it to fire sometimes when there are no regions.